docs/source/contributing/version-policy.md

```diff
@@ -16,10 +16,16 @@ This chapter describes the versions of software that Volto supports and how Volt
 
 (volto-generator-compatibility-with-volto-label)=
 
-## `volto-generator` compatibility with Volto
+## `@plone/volto-generator` compatibility with Volto
 
-`volto-generator` is a tool that generates a Volto project.
-The following table describes the compatibility between versions of `volto-generator` and Volto.
+```{deprecated} Volto 18.0.0
+`@plone/volto-generator` was deprecated in Volto 18.0.0.
+It is not maintained, and you should not use it.
+For new projects, use [Cookieplone](https://github.com/plone/cookieplone) instead.
```

Comment:
We should link to https://6.docs.plone.org/install/create-project-cookieplone.html here, not the Cookieplone repository.

Reply:
The whole sentence needs a rewrite. This syntax should work via Intersphinx for both here and `plone/documentation`.

```suggestion
For new projects, follow {doc}`plone:install/create-project-cookieplone` instead.
```

=>

For new projects, follow [Install Plone with Cookieplone](https://6.docs.plone.org/install/create-project-cookieplone.html) instead.